<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphEditorForm</class>
 <widget class="QWidget" name="GraphEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="graphScene_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Отображение графа</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="Graph::GraphEditView" name="graphScene" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="props_stackedWidget">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="mainProps_page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="Graph::GraphPropertyEditForm" name="propertyEditForm" native="true">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="objectProps_page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="ObjectPropertyEditorForm" name="objectPropertyEditForm" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Graph::GraphEditView</class>
   <extends>QWidget</extends>
   <header location="global">GraphScene/EditView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Graph::GraphPropertyEditForm</class>
   <extends>QWidget</extends>
   <header>graphpropertyeditform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ObjectPropertyEditorForm</class>
   <extends>QWidget</extends>
   <header location="global">GraphScene/ObjectPropertyEditorForm.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
